src/Controller/Api/FormController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use JMS\Serializer\Serializer;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Validator\ConstraintViolationList;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * @Route("/form")
+ */
+class FormController extends Controller
+{
+    /**
+     * @Route("/validate/{formType}", name="api_form_validate")
+     * @Method("POST")
+     */
+    public function validate(Request $request, Serializer $serializer, string $formType): Response
+    {
+        $form = $this->createForm($formType, null, ['csrf_protection' => false]);
+
+        $form->submit($params = $request->request->all()[$form->getConfig()->getName()], false)->isValid();
```

Comment:
why do you need `$params`

Reply:
to check each params send with validator